pkgs/os-specific/linux/libsmbios/default.nix

```diff
@@ -1,36 +1,38 @@
-{ stdenv, fetchurl, pkgconfig, libxml2, perl, autoreconfHook, doxygen }:
+{ stdenv, fetchFromGitHub, pkgconfig, autoreconfHook, libtool, gettext
+, libxml2, perl, doxygen }:
 
-let
-  version = "2.3.2";
-in
-stdenv.mkDerivation {
+
+stdenv.mkDerivation rec {
   name = "libsmbios-${version}";
+  version = "2.3.3";
 
-  src = fetchurl {
-    url = "https://github.com/dell/libsmbios/archive/v${version}.tar.gz";
-    sha256 = "0kvi36jrvhspyyq0pjfdyvzvimdn27fvbdpf429qm3xdmfi78y2j";
+  src = fetchFromGitHub {
+    owner = "dell";
+    repo = "libsmbios";
+    rev = "v${version}";
+    sha256 = "1cl5nb6qk8ki87hwqf9n1dd9nlhkjnlpdxlhzvm82za16gs7apkl";
   };
 
-  nativeBuildInputs = [ autoreconfHook pkgconfig ];
-  buildInputs = [ libxml2 perl doxygen ];
+  nativeBuildInputs = [ autoreconfHook doxygen gettext libtool perl pkgconfig ];
+  buildInputs = [ libxml2 ];
+
+  configureFlags = [ "--disable-python" "--disable-graphviz" ];
 
-  # It tries to install some Python stuff even when Python is disabled.
-  installFlags = "pkgpythondir=$(TMPDIR)/python";
+  enableParallelBuilding = true;
 
-  postInstall =
+  postInstall = 
```

Comment:
can you remove this trailing whitespace?

Reply:
Done